config/onlstm.conf

```diff
@@ -0,0 +1,20 @@
+include "defaults.conf"
+elmo = 0
+bidirectional = 0
+pretrain_tasks = "wsj"
+target_tasks = "wsj"
+do_pretrain = 1
+do_target_task_training = 1
+do_full_eval = 1
+allow_reuse_of_pretraining_parameters = 1
+word_embs = "scratch"
+sent_enc = "onlstm"
+d_hid = 1150  //default ON-LSTM paper hyperparameter
+n_layers_enc = 3  //default ON-LSTM paper hyperparameter
+dropout = 0.3  //default ON-LSTM paper hyperparameter
+d_word = 400  //default ON-LSTM paper hyperparameter
+max_seq_len = 70
+skip_embs = 0
+batch_size = 20
+optimizer = sgd
+lr = 1.0  //default in ON-LSTM paper was 30.0, changed because no repackaging of hidden state.
```

Comment:
I don't know what repackaging is. (I don't think it's a standard term.) Spell this out a bit.